app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -62,6 +62,14 @@ protected function create(StoreTenant $request)
 		$tenant->setOwner($user);
 		$tenant->owner->notify(new ActivateTenant);
 
+		$payStackCustomer = PayStack::createCustomer([
+			'first_name' => $request->firstname,
+			'last_name' => $request->lastname,
+			'email' => $request->email
+		]);
+
+		$tenant->fill(['paystack_id' => $payStackCustomer->customer_code]);
```

Comment:
```suggestion
		$tenant->update([
		'paystack_id' => Arr::get($payStackCustomer, 'data.customer_code',  data)
		]);
```

we need to either call `$tenant->save()` when we use `[fill](https://laravel.com/docs/7.x/eloquent#mass-assignment)` as changes aren't being persisted without it. using `update` calls save.

also there are lots of array [helpers](https://laravel.com/docs/7.x/helpers#method-array-get) that allow us to do kool things with arrays. Also i believe the paystack response is in the [data](https://paystack.com/docs/api/#customer-create) property.